sci-biology/fsl/fsl-5.0.8.ebuild

```diff
@@ -108,7 +108,8 @@ src_install() {
 	sed -i "s:\$FSLDIR/tcl:/usr/libexec/fsl:g" \
 		$(grep -l "\$FSLDIR/tcl" tcl/*) || die
 
-	dobin bin/*
+	exeinto /usr/share/${PN}/bin
```

Comment:
@jlec I actually believe that would make things much more difficult. FSL treats `$FSLDIR` like a sort of FSL-only subsystem. It is supposed to contain `$FSLDIR/bin` - but not only. Have a look at an example from the FSL sources:

```
29:FSLTCLSH=$FSLDIR/bin/fsltclsh
30:FSLWISH=$FSLDIR/bin/fslwish
52:FSLCONFDIR=$FSLDIR/config
53:FSLMACHTYPE=`$FSLDIR/etc/fslconf/fslmachtype.sh`
```